Exercises/NoModules/Chapter 05/Soln5_07.cpp

```diff
@@ -0,0 +1,51 @@
+// Exercise 5-7. Outputting product records & cost
```

Comment:
In my draft, 5-7 was 
"Exercise 5-7. Repeat Exercise 5-6, but this time remove the multiples of 7 and 13 from the vector<> container before you output the first element."
But I guess you removed that exercise.